buildSrc/src/main/groovy/nva-fetch-doi.java-common-conventions.gradle

```diff
@@ -64,13 +64,22 @@ checkstyle {
 }
 
 tasks.withType(Checkstyle) {
+    exclude { '**/generated/*' }
     reports {
         xml.enabled false
         html.enabled true
         html.stylesheet rootProject.resources.text.fromFile('config/checkstyle/checkstyle-simple.xsl')
     }
 }
 
+tasks.withType(Pmd) {
+    exclude { '**/generated/*' }
```

Comment:
Not necessary…see comment on subproject build.gradle

Reply:
I don't get why think this is unnecessary. Somewhere we have to tell PMD, Checkstyle and JaCoco to ignore those.
I think its best done at the root level.